code/iaas/logic/src/main/java/io/cattle/platform/process/lb/LoadBalancerAddRemoveHost.java

```diff
@@ -19,22 +18,22 @@
 @Named
 public class LoadBalancerAddRemoveHost extends AbstractObjectProcessHandler {
 
-    JsonMapper jsonMapper;
+    @Inject
     GenericMapDao mapDao;
 
     @Override
     public String[] getProcessNames() {
         return new String[] { LoadBalancerConstants.PROCESS_LB_ADD_HOST, LoadBalancerConstants.PROCESS_LB_REMOVE_HOST };
-
     }
 
     @Override
     public HandlerResult handle(ProcessState state, ProcessInstance process) {
         LoadBalancer lb = (LoadBalancer) state.getResource();
-        Long hostId = DataAccessor.fromMap(state.getData()).
+        long hostId = DataAccessor.fromMap(state.getData()).
```

Comment:
hostId is set with required=true in the API (addRemoveLoadBalancerHostInput.json)

"resourceFields" : {
        "hostId" : {
            "type": "reference[host]",
            "nullable": false,
            "required": true
        }

Unless there is any other entry point for the Process rather than API, it should be fine I think.